layers/lldp.go

```diff
@@ -37,6 +37,10 @@ type LinkLayerDiscoveryValue struct {
 	Value  []byte
 }
 
+func (c *LinkLayerDiscoveryValue) len() int {
```

Comment:
going to merge as is, but is this function actually used?  if not, please remove in future pull.